### PR TITLE
Adding background colour behind loading image

### PR DESCRIPTION
**Pull request by tomgco**
This makes the experience on mobile nicer as the image loads from a
colour associated to the image
screenshot from 2016-06-30 16-10-44

### CODE REVIEW

**Comment by nelsonic** [comment]
@tomgco did you _manually_ detect which color to use for each image?

**Comment by hassy** [comment]
Nice UX touch @tomgco :)

**Comment by jackcarlisle** [comment]
@tomgco does this mean we'll only be using one image _(`heroImageUrl[0]`)_ as the header?

File: lib/hero-image-header/index.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ export default class Header extends Component {
   }
   render () {
     const wrapperStyle = {
-      backgroundImage: `url(${heroImageUrl})`
+      backgroundImage: `url(${heroImageUrl[0]})`,
```
Comment (nelsonic) [inline lib/hero-image-header/index.js]: > @jackcarlisle's point regarding the hero image being _hard-coded_ ... 
My _understanding_ of this code is that the `0` index is the url and `1` is the `background-color` ...

Comment (tomgco) [inline lib/hero-image-header/index.js]: Correct @nelsonic

**Comment by tomgco** [comment]
> @tomgco did you manually detect which color to use for each image?

Yes, manually plugged in the best colour for it, we could have some funky stuff in here later, but seeming we only had < 10 images, thought why not.

**Comment by jackcarlisle** [comment]
@tomgco my bad, I misunderstood it!

**Pull request merged**